sys/include/trickle.h

```diff
@@ -12,12 +12,18 @@
 /**
  * @defgroup sys_trickle Trickle Timer
  * @ingroup sys
+ * @brief   Implementation of a generic Trickle Algorithm (RFC 6206)
+ *
+ * @see https://tools.ietf.org/html/rfc6206
+ *
  * @{
+ *
  * @file
- * @brief   Implementation of a generic Trickle Algorithm (RFC 6206)
+ * @brief   Trickle timer interface definition
  *
  * @author  Eric Engel <eric.engel@fu-berlin.de>
  * @author  Cenk Gündoğan <cenk.guendogan@haw-hamburg.de>
+ *
```

Comment:
good question, or better: normally I add an empty line between last `@author` and closing either a group `@}` and/or ending a comment `*/`. **BUT** its not consistent in RIOT nor did I (see this PR) stick to it 😬 

What do you think, w/ or w/o empty line?